stac_api/runtime/src/links.py

```diff
@@ -46,12 +46,16 @@ def inject_item(self, item: Item) -> None:
         item_id = item.get("id", "")
         item["links"] = item.get("links", [])
         if self.tiler_href:
-            item["links"].append(self._get_item_map_link(item_id))
-            item["assets"]["rendered_preview"] = self._get_item_preview_link(item_id)
+            item["links"].append(self._get_item_map_link(item_id, self.collection_id))
+            item["assets"]["rendered_preview"] = self._get_item_preview_link(
+                item_id, self.collection_id
+            )
 
-    def _get_item_map_link(self, item_id: str) -> Dict[str, Any]:
-        qs = self.render_config.get_full_render_qs(self.collection_id, item_id)
-        href = urljoin(self.tiler_href, f"stac/map?{qs}")
+    def _get_item_map_link(self, item_id: str, collection_id: str) -> Dict[str, Any]:
+        qs = self.render_config.get_full_render_qs()
+        href = urljoin(
+            self.tiler_href, f"collections/{collection_id}/items/{item_id}/map?{qs}"
```

Comment:
so the new href is `collections/.../items/.../map` as opposed to `stac/map`? Sorry I don't already know this but how could I verify this change?

Reply:
You could try out this link: https://dev.openveda.cloud/api/raster/collections/geoglam/items/CropMonitor_202403/map?bidx=1&assets=cog_default&unscale=False&colormap=%7B%221%22%3A+%5B120%2C+120%2C+120%5D%2C+%222%22%3A+%5B130%2C+65%2C+0%5D%2C+%223%22%3A+%5B66%2C+207%2C+56%5D%2C+%224%22%3A+%5B245%2C+239%2C+0%5D%2C+%225%22%3A+%5B241%2C+89%2C+32%5D%2C+%226%22%3A+%5B168%2C+0%2C+0%5D%2C+%227%22%3A+%5B0%2C+143%2C+201%5D%7D&max_size=1024&resampling=nearest&return_mask=True